client/src/pages/Restaurant/RestaurantView/index.js

```diff
@@ -18,6 +18,7 @@ export default function RestaurantView({ restaurant, foods }) {
       <Header restaurant={restaurant} />
       <RestaurantInfo restaurant={restaurant} />
       <BudgetFilter foodsInBudget={foodsInBudget} budget={budget} setBudget={setBudget}/>
+      <RestaurantMap latitude={40.768538} longitude={-73.964741}/>
```

Comment:
We have hunter college coordinates in a constants folder. Forgot where it is, but use it instead of hardcoding it like this